pkg/serviceaccounttoken/secret_cleaner_test.go

```diff
@@ -0,0 +1,292 @@
+package serviceaccounttoken
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/rancher/rancher/pkg/features"
+	"github.com/rancher/wrangler/v3/pkg/generic"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestCleanServiceAccountSecrets(t *testing.T) {
+	var secrets []*corev1.Secret
+	for i := range 7 {
```

Comment:
I would move this in a dedicated var with a name, with also the others fixed numbers. Something like

```go
var (
    totalSecrets   = 7
    cleanedSecrets = 5
)
```